src/pages/references/changelog.md

```diff
@@ -27,6 +27,33 @@ contributors:
 
+### Important notes on Script Runtime APIs
+
+* Intermittent issues
+  * Auto reload of the add-on when a change is detected sometimes fails to work properly. This can result in changes to the UI HTML not being reflected, but can also cause the connection between the panels UI and the script runtime to not be properly initialized. If you encounter this situation, a reload of the add-on from the developer panel will usually resolve the issue. We're working on a fix.
+  * It's occasionally possible to run into a race condition where the communications bridge between the two contexts (Panel vs script runtime) is not set up in time. That may cause an intial click in your panel to fail invoking your script code. We're working on a fix for this.
+* Performance
+  * The script runtime uses QuickJS to ensure that the code is properly sandboxed. This does mean that performance for some operations may be significantly slower than you might expect. In particular, you may see warnings on the console similar to `Detected a possible stutter. Excessive ECS Frame duration of ## ms` when adding lots of shapes or performing calculations. The higher this number is the more likely the user is going to perceive your add-on as slow and may think that your add-on has crashed. You may want to consider splitting your work into multiple frames (using `setTimeout`) in order to reduce the frequency of these messages. Note that currently it can be hard to avoid these messages even when adding only a few shapes to the document at time. A few milliseconds isn't a big deal, but you should try to avoid frames that take more than 150ms, which may start to cause other warnings to appear, and may cause users to feel like your add-on is not responsive.
+  * If your code takes more than about 150-160ms to process, you may see warnings from the co-editing service in the console (`LECCoeditingService: Invalid sequenceNumber`). The cause for this is being researched and will be addressed in a future release. To reduce the number of times you see this error, reduce the length of time your script takes in a single editing frame — use `setTimeout` if necessary to batch your work.
+* Common pitfalls
+  * If you split your work on a document over multiple frames using `setTimeout`, be sure to protect against reentrancy, otherwise you may end up corrupting the user's undo stack. You should disable elements on the panel UI that could allow the user to execute your code before it is complete and then re-enable those elements when the code is done. The issue will be fixed in a future release. 
+  * Executing `await runtime.apiProxy("script")` from your script runtime code will do nothing, and nothing will be logged on the console.
+  * When adding shapes to a document, you should take care to ensure that the content is at least partially visible on the page. Otherwise the user may not have a way to edit it easily in the future.
```

Comment:
Was this a problem in our testing this week? By default content would be positioned in the upper left corner of the page – so it seems like this would only crop up if an add-on was trying to do more custom positioning manually and had a bug.

Reply:
It wasn't a problem for us, but I saw it happen in one of the demos from the second hackathon. I'll have to go look it up again.